meta_capstone/ViewControllers/InGameViewController.m

```diff
@@ -165,37 +172,47 @@ - (void)printSquareArray : (NSMutableArray *)array {
 
 //initialize board ui
 - (__kindof UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cellForItemAtIndexPath:(NSIndexPath *)indexPath {
+    
+    self.xIndex = (int)([indexPath item] % 10);
```

Comment:
what are you doing here?

Reply:
indexpath is a number between 0-99 for my collection view, and the way the cell indexpath lines up with the indexes is the first digit -> yIndex and second digit -> xIndex. i realized reloading collection view doesn't happen from 0 to 99 in a row, so i had to change the way I was getting indices to be related to the indexpath.